.eslintrc.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
     }],
 
     // enforce license header (todo: improve plugin to support patterns for multi-lines)
-    'header/header': [2, 'block', ['',
+    'header/header': [2, 'block', ['*',
```

Comment:
This is not correct, it turns the license comment from a normal block comment `/*` into a JSDoc comment `/**`

Reply:
i put this because typescript compiler only considers JSDoc, so it will remove most of the license headers